internal/controllers/control/jobs.go

```diff
@@ -442,6 +442,22 @@ func (r *CassandraTaskReconciler) refreshDatacenter(ctx context.Context, dc *cas
 		}
 	}
 	return ctrl.Result{RequeueAfter: JobRunningRequeue}, nil
+
+}
+
+// Compaction functionality
+
+func TSReloadAsync(nodeMgmtClient httphelper.NodeMgmtClient, pod *corev1.Pod, taskConfig *TaskConfiguration) (string, error) {
```

Comment:
What is this function? It doesn't do what it says.

Reply:
This wasn't ready for review, but let's go anyway... I removed this, we don't need it.